index.html

```diff
@@ -5,12 +5,72 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="icon" href="Delicious-cupcake-PNG-1.png" type="image/icon type">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" rel="stylesheet" />
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header>
+        <div class="logo">
+            <img src="Delicious-cupcake-PNG-1.png" alt="logo">
+            </div>
+            <nav>
+                <ul>
+                    <input src='checkbox' id='check' />
```

Comment:
if you change `src` attribute to `type` it will behave like a checkbox

Reply:
oh, I'll chenged it as soon as possible thank you for your attention Jon